roop/core.py

```diff
@@ -198,10 +201,14 @@ def start() -> None:
     seconds, probabilities = predict_video_frames(video_path=roop.globals.target_path, frame_interval=100)
     if any(probability > 0.85 for probability in probabilities):
         destroy()
-    update_status('Creating temp resources...')
-    create_temp(roop.globals.target_path)
-    update_status('Extracting frames...')
-    extract_frames(roop.globals.target_path)
+    # temp directory with frames is existed and all frames are extracted (or some already processed)
+    if state.is_exists(roop.globals.target_path) or state.unprocessed_frames_count(roop.globals.target_path) == detect_fc(roop.globals.target_path):
+        update_status(f'Temp resources for this target already exists with {state.processed_frames_cnt} frames processed, continue processing...')
```

Comment:
Please do not use shortcuts in our code... there is no idx, cnt or err allowed...

Reply:
done